electives/18660.md

```diff
@@ -0,0 +1,47 @@
+# 18-660: Oprimization
+
+| Category | Difficulty |
+|:-:       | :-:        |
+| HW       | 4          |
+| Exams    | 3          |
+| Project  | varies     |
+
+In Spring 2022, this course was taught by Prof. Qu.
```

Comment:
It would be good to mention what this class is good for doing, like what careers or future classes do people usually take after this class / along with this class?